Etiquette/CODE_REVIEW.md

```diff
@@ -0,0 +1,110 @@
+* Self-review your changes, adding comments where you think you can get questions (it may worth adding these as code or documentation comments). Tip: if you spot small issues you can use GitHub suggestions and apply them right away
+* Keep your pull requests small, ideally they shouldn't be more than **800 additions/deletions**
+* Create a draft PR for work that is in progress, but only do that if you seek for opinions on your work
+* Mark PR with the "Blocked" label if it has any dependency that should stop it from being merged (it's fine for pull request be both "blocked" and "ready for review", but it shouldn't be "ready for review" and "wip")
+* Give your team mates a reasonable time to review your changes, if the change is critical use "Top priority" label and post it in the team channel
+* Do not leave stale pull requests, seek for help/actions or close them (make sure you don't delete your local branch if you may need it later). If you find one that appears to have been stale for a week, please warn the author. After two weeks it can be closed with a note after a discussion of its current status with the author.
+* It's our common effort to push product forward and we all do equally valuable work. Keep that in mind when creating a PR: try to review some of other's PRs, and avoid having more than X PRs opened at the same time
+
+
+Reviewing a Pull Request
+--------
+
+* Accept that many programming decisions are opinions. Discuss tradeoffs, which
+  you prefer, and reach a resolution quickly. There are always multiple ways of doing things, and they may be equally acceptable
+* Focus not on finding flaws in the code (no one writes perfect code) but on understanding the change in the first place
+* Encourage things you like, try not only comment on what you don't like
+* Before suggesting to change code think if it brings any real value or if it is just your personal preference. Is this code something you can live with or not?
+* Do not try to make everything perfect (it never will be and otherwise we'll never deliver anything), but do not compromise on quality
+* Do not comment if you don't have any suggestions to address your comment
+* Use GitHub suggestions as much as possible
+* If you feel strong against the change - request changes and explain why you are doing that. Do not request changes for typos, missing documentation comments or code style violations, trust your colleagues to address them.
+* Approve changes if you are confident about them. 
+* If you are the first reviewer to approve the changes add the label "Needs one reviewer" so that others can see which PRs already have a feedback and which don't have any yet.
+* Comment on the changes if something is not clear to you and needs further clarification or actions
+* If you are not familiar with the domain ask for more context (better in person) rather than skipping to something more familiar
```

Comment:
```suggestion
* If you are not familiar with the domain ask for more context (better directly to the author) rather than skipping to something more familiar
```